api/generateJWT.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/sh
+
 docker-compose exec php sh -c '
+    export HTTP_PROXY='$2'
+    export HTTPS_PROXY='$3'
```

Comment:
You're exporting unnecessary variables in the most of cases

Reply:
I have been debugging the installation for 5 days non stop with @jenaye so let me tell you this is not unnecessary at all. Try to install it without those exports and the apk add command will fail. This will also happed for every commands related to composer, npm and cypress.